chef.py

```diff
@@ -411,19 +412,50 @@ def process_sim_html(content, destpath, **kwargs):
     # remove online links from "about" modal
     content = content.replace("getLinks:function(", "getLinks:function(){return [];},doNothing:function(")
 
+    # setting up the query parameters for cases in which it works
+    regex = r"(this\.getAllForString\([^,]+,[^w]*)(window\.location\.search)(\s*\))"
+    replacement = r"\1window.location.search === '' ? '?allowLinks=false&disableFullscreen' : window.location.search\3"
+    content = re.sub(regex, replacement, content)
     soup = BeautifulSoup(content, "html.parser")
 
+
     for script in soup.find_all("script"):
         # remove Google Analytics and online image bug requests
         if "analytics.js" in str(script):
             script.extract()
+
         if 'phetWebsite' in str(script):
             # remove menu options that link to online resources
+            # fallbacks in case the above regex work fails
             script.string = re.compile('tandem: e.createTandem\(\"screenshotMenuItem\"\),').sub("", script.string)
             script.string = re.compile('tandem: e.createTandem\(\"fullScreenMenuItem\"\),').sub("", script.string)
-            script.string = re.compile('tandem: e.createTandem\(\"screenshotMenuItem\"\)').sub("", script.string)
             script.string = re.compile('string!JOIST/menuItem.reportAProblem').sub("", script.string)
-
+            script.string = re.compile('string!JOIST/menuItem.phetWebsite').sub("", script.string)
+
+    clean_title = kwargs.get("sim_title").replace(" ", "-").replace("'", "-")
+
+    ##############
+    # DEBUG HELP #
+    ##############
+    """
+    Set this to True to dump all sims to processed_sims directory
+    I wanted to make this a command line option but ricecooker does some parsing that doesn't 
+    permit flags without assigned values (e.g. --dump-sims has to be --dump-sims=<anything>)
+    Once the sims are processed, you can run `python -m http.server <port>` to test them in your 
+    browser
+    """
+    if True:
```

Comment:
Maybe just have this check an env var instead? `DUMP_SIMS=True python chef.py`

Reply:
@rtibbles good idea - done